src/Tasks/Common/Resources/xlf/Strings.it.xlf

```diff
@@ -4,12 +4,12 @@
     <body>
       <trans-unit id="AddResourceWithNonIntegerResource">
         <source>NETSDK1076: AddResource can only be used with integer resource types.</source>
-        <target state="translated">NETSDK1076: è possibile usare AddResource solo con tipi di risorsa integer.</target>
+        <target state="new">NETSDK1076: AddResource can only be used with integer resource types.</target>
```

Comment:
Bug here in xliff-tasks. I've seen it a few times now and really need to get to the bottom of it. Looks like a race condition. The translations got reverted.

Reply:
i will revert this file and run generation again

Reply:
done